docs/explanation/index.rst

```diff
@@ -3,7 +3,21 @@
 Explanation
 ===========
 
+The explanation material in this section expounds on Charmcraft's high-level concepts,
+helping you understand how everything fits together. It speaks to the rationale behind
+Charmcraft's design, features, and implementation.
+
+
+Cryptography
+------------
+
+Charmcraft and its external libraries use cryptographic tools for fetching files,
+communicating with local processes, and storing user credentials. These tools are
+detailed in :ref:`explanation-cryptographic-technology`.
```

Comment:
I think the pattern we want here, visually, is for links to be on their own lines. When there's a list of links, the parallelism will make more sense. I demonstrated this in the Rockcraft PR.

```suggestion
communicating with local processes, and storing user credentials.

:ref:`explanation-cryptographic-technology`
```